### PR TITLE
Chapters design

### PR DESCRIPTION
**Pull request by duarte**
Design tweaks to chapters

### CODE REVIEW

**Comment by duarte** [comment]
@gidsg can you please see commit [f447891](https://github.com/guardian/docs-interactive-template/pull/39/commits/f44789151acc254fea901e06916b52569a99f86e) specifically, I changed the first element passed on to `performPlayActions` because the video wasn't getting expanded at all, possibly just a regression from before — working now tho.

**Comment by mr-mr** [comment]
Why are you hiding the chapters before playing? I thought the plan was to see where people were clicking to watch the video - the main play or the chapters?

File: src/css/_docs.scss [hunk 1]
```diff
@@ -405,38 +405,48 @@
...
+    }
   }
   li {
     cursor: pointer;
     vertical-align: top;
     display: inline-block;
     @include fs-textSans(4);
     white-space: pre-wrap;
-    width: 150px;
+    /* width: 150px; */
```
Comment (mr-mr) [inline src/css/_docs.scss]: Remove commented out lines

**Comment by duarte** [comment]
Yeah this'll eps the initial page much cleaner though. I see how we can't test that with them expanding on click, but think the chapters still really make sense to have tho?

**Comment by mr-mr** [comment]
I don't understand your comment. Of course we're keeping the chapters work. What doesn't make sense to me is hiding them. Cleaner or not, I want to see clicks to watch video, whether that comes from the main button or the chapters.

**Comment by gidsg** [comment]
OK, so If we are interested in measuring if people use the chapters initially, rather than revealing them on play next steps would be:
@duarte revert  89d72e5
@mr-mr (with help from @gidsg) add tracking, so we can tell if people use the chapter buttons to start the video.

@duarte would you be OK with this as a next step?

**Comment by mr-mr** [comment]
So what's the plan then? Are we going with @gidsg suggestion or going ahead as before?

**Comment by duarte** [comment]
Hey — it's more important that we get the design right, especially on the initial load, than to measure people clicking on chapters directly vs play button. I'd like to merge and go ahead with this change.

**Comment by mr-mr** [comment]
Fair enough. I thought we'd discussed and  agreed to show the chapters on load for the first documentary. Go ahead and merge though if you think the design is right.

**Comment by mr-mr** [comment]
:+1

**Comment by mr-mr** [comment]
Well that doesn't thumbs up on mobile.
I did a refactor on the inner htmls in main.js
Might be worth doing a pull and rebase of master just to make nothing is going to break. 

Though it looks like it's fine.

*thumbs up.

**Comment by gidsg** [comment]
👍 ⛵

**Pull request merged**